saleor/product/models.py

```diff
@@ -193,21 +194,21 @@ def __repr__(self) -> str:
 
 class ProductsQueryset(models.QuerySet):
     def published(self, channel_slug: str):
-        today = datetime.date.today()
+        today = timezone.now()
```

Comment:
Why we don't use `datetime.datetime.now(pytz.UTC)` here? Shouldn't we use the same method to get the current time?